storeshare_api/app/models/contract.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Contract < ApplicationRecord
+  belongs_to :listing
```

Comment:
should price be stored directly in contract? 
(i.e. a listing's price might change over time, but the price within an older contract should stay the same)

Reply:
added 👍